mailserver/mailserver.go

```diff
@@ -535,7 +542,7 @@ func (s *WMailServer) processRequestInBundles(
 		lastEnvelopeHash = envelope.Hash()
 		processedEnvelopes++
 		envelopeSize := whisper.EnvelopeHeaderLength + uint32(len(envelope.Data))
-		limitReached := limit != noLimits && processedEnvelopes == limit
+		limitReached := processedEnvelopes == limit
```

Comment:
it doesn't matter if we received less then a limit?

Reply:
what do you mean exactly?
If there are no more envelopes (less than the limit) we leave the loop, and never reach this condition, is this what you were thinking?

Reply:
yeah, sorry. i didn't realize that this part was reading from db until limit is reached..